web/src/components/CopyText.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <v-text-field
+    ref="textField"
+    v-model="text"
+    readonly
+    :success-messages="messages"
+    v-bind="$attrs"
```

Comment:
Is this used so that arbitrary props can be passed through this component to the `v-text-field` (like `outlined`)?

Reply:
Correct. I left a comment this at the top of the component script, but you can't really leave comments on HTML templates.